python/src/numerous/utils.py

```diff
@@ -1,12 +1,30 @@
 """Miscellaneous utilities."""
 
-from typing import TypeVar
+import base64
+import json
+from typing import Dict, TypeVar, cast
 
 
 class _MissingType:
     pass
 
 
 MISSING = _MissingType()
+DECODED_JSON_NOT_DICT = "Decoded JSON is not a dictionary"
 
 AppT = TypeVar("AppT")
+
+
+def dict_to_base64(input_dict: dict[str, str]) -> str:
```

Comment:
Maybe this could be in a `jsonbase64.py` module instead of in `utils.py`? The existing things in `utils.py` are more related to the app engine stuff.

Reply:
And I mean this is for all things you added to the file. :smile: I also think you can just inline `DECODED_JSON_NOT_DICT`, instead of storing it in a variable.

Reply:
@artem-chupryna Instead of inlining you could move the variable. Having it in the module-level scope when it is only used once is a bit confusing.